Makefile

```diff
@@ -74,3 +74,6 @@ endif
 	go get github.com/ElrondNetwork/arwen-wasm-vm/cmd/arwen@$(shell cat go.mod | grep arwen-wasm-vm | sed 's/.* //')
 	go build -o ${ARWEN_PATH} github.com/ElrondNetwork/arwen-wasm-vm/cmd/arwen
 	stat ${ARWEN_PATH}
+
+cli-docs:
```

Comment:
What is the real usage of auto generated help .md files that contain the exact output of the binary ran with -help flag?

Reply:
They will be referenced from our docs: https://docs.elrond.com/validators/node-cli

Reply:
ok, makes sense